vuepress/docs/tutorials/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md

```diff
@@ -0,0 +1,170 @@
+# Use Blueprint generated plugin and microfrontend without a bundle
+
+## Purpose
+
+In this tutorial you will learn how to use the plugin and microfrontend generated using the Entando Blueprint
+in a running cluster without the need to include the components in a bundle 
+
+## Requirements
+
+-   A microservice built with the Entando Component Generator
+
+-   Node and NPM are installed on your machine (use LTS version)
+
+-   Docker is installed on your machine and you are able to upload images to docker-hub or an image repository of your
+    choice
+
+-   An Entando Cluster where to deploy the Entando Plugin with a working instance of Entando App Builder 
+    and Entando App Engine
+
+## Steps
+
+## 1. Generate an entity using the Blueprint
+
+In order to include auto-generated micro frontends to your bundle you
+will need to use the `jhispter entity` generator. Let’s assume we want
+to generate a simple *Conference* entity.
+
+    jhipster entity Conference
+
+Let’s use these options:
+
+-   One field of type `String` called `name` with no validation
+
+-   No relationships with other entities
+
+-   REST controller should use the repository directly
+
+-   No pagination for the entity
+
+## 2. Populate the bundle with the micro frontends
+<span id="section2"></span>
+
+To populate the bundle with the generated micro frontends, run the
+`./buildBundle.sh` script or use the npm.
+
+    npm run populate-bundle
+
+> **Important**
+>
+> The bundle population with the micro frontends requires some time to
+> be processed. You should be able to follow the progress of the
+> operation on screen.
+
+## 3. Generate a docker image for your microservice
+
+JHipster uses the JIB Maven plugin to generate a docker image for your
+microservice.
+
+By default the output image will use the name of your application and
+the `0.0.1-SNAPSHOT` tag, eg. `entando/jhipster@0.0.1-SNAPSHOT`.
+
+    ./mvnw -Pprod clean package jib:dockerBuild [-Djib.to.image=<your-image-name]
+
+> **Note**
+>
+> By default, the organization used for the image is `entando` but you
```

Comment:
I think we can mention here that in the new wizard, user is asked for an `organization` where to publish the docker image. That organization is set by default to `entando` but the user can override it quite easily. Using their custom organization solves issues like "I'm not authorized to publish a docker image in the `entando` organization on dockerhub", or "I've built my custom image but the microservice I'm deploying is still referencing the `entando` organization"